OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -464,7 +464,7 @@ public virtual int GetProductionCost(ActorInfo unit)
 				return 0;
 
 			var modifiers = unit.TraitInfos<IProductionCostModifierInfo>()
-				.Select(t => t.GetProductionCostModifier(techTree, Info.Type));
+				.Select(t => t.GetProductionCostModifier(techTree, Info.Type, self.World, unit));
```

Comment:
Style nit: our usual conventions are to pass `World`, then `ActorInfo`, then the rest of the args.

Reply:
Flipped it and renamed the variable to `actorInfo`.